roles/tendrl-performance-monitoring/tasks/main.yml

```diff
@@ -21,8 +21,33 @@
   with_items:
     - regexp: '^etcd_connection:*'
       line: "etcd_connection{{ ':' }} {{ groups['usm_server'][0] }}"
-    - regexp: '^api_server_addr:*'
-      line: "api_server_addr{{ ':' }} {{ groups['usm_server'][0] }}"
+    - regexp: '^etcd_port:*'
+      line: "etcd_port{{ ':' }} 2379"
+    - regexp: '^time_series_db_server:*'
+      line: "time_series_db_server{{ ':' }} {{ ansible_eth0['ipv4']['address'] }}"
```

Comment:
Because in documentation was stated that there should be ip address of system on which time-series db(graphite) is installed.